Jari vddr patch 1

JariVddr opened the pull request:
Adding Biodiversity Metric

anaraqueltomas commented:
@JariVddr great effort!!
Thanks very much for trying out the 'geek way' of adding tools! I've just added some changes, mostly due to incorrections in our tools_template.json (so, your pull request is already generating some corrections in this workflow!).

But I still need your input regarding `data_provenance`, `development_process` and `documentation_contribute`. These are required, could you provide an answer for those? Even if it is a "Don't know". Then I'll be able to accept the new tool! :)

anaraqueltomas commented:
There's only this pull request with the biodiversity-metric.json. Can you check, or try to redo the other pull request?
I've already added the tools that you filled in the form as well.

We are trying to improve the way to add new tools, which is not the most very user friendly for now.
The recommendation is to use the [tools_template.json](https://github.com/euraf/AF-tools-database/blob/main/tools/tools_template.json), and base the responses on the information available in the [tools_form.json](https://github.com/euraf/AF-tools-database/blob/main/tools/tools_form.json). I'm not sure if this was the way you did it the first time...

Let me know when you have added the answers to the three missing questions (`data_provenance`, `development_process` and `documentation_contribute`).

JariVddr commented:
I think I accidently created the second pull request within my own branches instead of to the base repository. It should be okay now I think: I changed the three biodiversity metrics inputs to 'don't know' as well!

***@***.***

From: Ana Tomás ***@***.***>
Sent: Wednesday, 10 January 2024 21:12
To: euraf/AF-tools-database ***@***.***>
Cc: Jari Vandendriessche ***@***.***>; Mention ***@***.***>
Subject: Re: [euraf/AF-tools-database] Jari vddr patch 1 (PR #2)


There's only this pull request with the biodiversity-metric.json. Can you check, or try to redo the other pull request?
I've already added the tools that you filled in the form as well.

We are trying to improve the way to add new tools, which is not the most very user friendly for now.
The recommendation is to use the tools_template.json<https://github.com/euraf/AF-tools-database/blob/main/tools/tools_template.json>, and base the responses on the information available in the tools_form.json<https://github.com/euraf/AF-tools-database/blob/main/tools/tools_form.json>. I'm not sure if this was the way you did it the first time...

Let me know when you have added the answers to the three missing questions (data_provenance, development_process and documentation_contribute).

-
Reply to this email directly, view it on GitHub<https://github.com/euraf/AF-tools-database/pull/2#issuecomment-1885646638>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BEGNCLI2PL4EETJPPNTTPL3YN3Y2ZAVCNFSM6AAAAABBOCYFVCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQOBVGY2DMNRTHA>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>
